### PR TITLE
Fix phase field tutorial

### LINKED ISSUE #23286: RunTime is not a registered object
## Bug Description
I am running phase-field module example " [Fe-Cr Phase Decomposition](https://mooseframework.inl.gov/modules/phase_field/Tutorial.html)". At the 2nd step, it always reports error RunTime is not a registered object even if I modified ALL-MODULES=yes in Makefile and recompiled the project.

## Steps to Reproduce
I used the same input file as the 2nd step "Make a faster model" (https://mooseframework.inl.gov/modules/phase_field/Tutorial/Step2.html)

## Impact
I am new uses of moose and I have to conduct phase-field simulation in my project. This really slows down my learning process.

### PR DESCRIPTION
**Pull request by dschwen**
Closes #23286

### CODE REVIEW

File: modules/phase_field/doc/content/modules/phase_field/Tutorial/Step2.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Parsed materials allow MOOSE to take the derivatives of the input functions auto
     function = 'eV_J*d*(A*c+B*(1-c)+C*c*log(c)+D*(1-c)*log(1-c)+
```
Comment (GiudGiud) [inline modules/phase_field/doc/content/modules/phase_field/Tutorial/Step2.md:25]: deprecated

**Review by GiudGiud — APPROVED** [review]
net fix but I would recommend switching to snippets

**Comment by dschwen** [comment]
I don't have a charge number for this.

**Comment by GiudGiud** [comment]
Yup that's fair

**Pull request merged**